src/observability/kubearmor.go

```diff
@@ -280,12 +281,23 @@ func GetKubearmorSummaryData(req *opb.Request) ([]types.SysObsProcFileData, []ty
 				Count:       uint32(ss.Count),
 				UpdatedTime: t.Format(time.UnixDate),
 			})
+		} else if ss.Operation == "Syscall" {
```

Comment:
constants exists for these "Syscall" and other strings ... Lets use those.

Reply:
Thank you for your review and suggestions. There are no existing constants found. I have added 4 new constants for operations and made the changes in respective fields.